### DISCUSSION TITLE
fixture argument `params` is not working

### DISCUSSION THREAD

**Discussion by devrimcavusoglu** [pytest-dev/pytest#10180] [Q&A] [answered]
I intended to use the `params` argument of fixtures, when I created my fixture I get an error.

```python
import pytest

valid_list = ["a", "b", "c", "d"]

@pytest.fixture(scope="module", params=[("a","b"), ("c", "d")])
def my_fixture(input_params):
    x, y = input_params.param
    assert x in valid_list and y in valid_list
```

what I get is,

```python
E       fixture 'input_params' not found
>       available fixtures: ...
>       use 'pytest --fixtures [testpath]' for help on them.
```

The thing is indeed `input_param` is not intended to be a fixture, it should be `("a","b")` and `("c", "d")` in two runs respectively.

**Comment by RonnyPfannschmidt** [comment, marked as answer]
as per [documentation](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#parametrizing-fixtures), the argument name is request, and `request.param` is where you find the data
